teku/src/main/java/tech/pegasys/teku/cli/slashingprotection/ExportCommand.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.slashingprotection;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import org.apache.logging.log4j.util.Strings;
+import picocli.CommandLine;
+import tech.pegasys.teku.cli.options.DataOptions;
+import tech.pegasys.teku.data.SlashingProtectionExporter;
+import tech.pegasys.teku.logging.SubCommandLogger;
+import tech.pegasys.teku.util.cli.PicoCliVersionProvider;
+
+@CommandLine.Command(
+    name = "export",
+    description = "Export slashing protection database in Minimal format.",
```

Comment:
```suggestion
    description = "Export slashing protection database in minimal format.",
```

Reply:
you sure? minimal is the name of the format...

Reply:
Docs team might overrule me but even with it being the name of the format it's not a proper noun so doesn't deserve a capital.